typescript/nuxtjs-prisma-nuxt-module/nuxt.config.ts

```diff
@@ -0,0 +1,8 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+export default defineNuxtConfig({
+  modules: ['@prisma/nuxt'],
+  prisma: {
+    autoSetupPrisma: true,
```

Comment:
let's link to our docs for these configs (if we don't have docs for them, then we should add them)

Reply:
@jharrell I added a dib link temporarily and will update it once the docs is out.